twirperrors/adapter.go

```diff
@@ -0,0 +1,93 @@
+package twirperrors
+
+import (
+	errors "github.com/segmentio/errors-go"
+	"github.com/twitchtv/twirp"
+)
+
+func Adapt(err error) (error, bool) {
```

Comment:
Should add godoc maybe? Not as important though since most people will never use this directly.

Reply:
👍 done!